pkg/utils/utils.go

```diff
@@ -413,3 +413,25 @@ func GetAKONamespace() string {
 	}
 	return AKO_DEFAULT_NS
 }
+
+func GetTokenFromRestObj(robj interface{}, ctrlAuthtoken string) (string, bool) {
```

Comment:
takes response object from /api/user-token and return with matching user token id (for deletion) and boolean that determines if refresh needs to performed
in case matching token is not found, it returns a no token id and true